scrypto/Cargo.toml

```diff
@@ -26,4 +27,9 @@ alloc = ["hashbrown", "hex/alloc", "sbor/alloc", "scrypto-abi/alloc", "scrypto-d
 trace = ["scrypto-derive/trace"]
 
 # Enable serde derives
-serde = ["sbor/serde", "scrypto-abi/serde"]
\ No newline at end of file
+serde = ["sbor/serde", "scrypto-abi/serde"]
+
+[profile.release]
+# Enable checking of overflow in Scrypto
+# Do not remove this as all scrypto code depends on overflow safe operations.
+overflow-checks = true
```

Comment:
Because the profile is chosen based on the root Crate that's actually compiled, we need to (also) add this to the template for Scrypto packages, as that's the root source of compilation for DApps